### DISCUSSION TITLE
How to create an .exe file installer instead of a .msi file?

### DISCUSSION THREAD

**Discussion by Fiinleyyy** [oleg-shilo/wixsharp#1552] [Q&A] [answered]
The question is quite simple als the title suggests. Right now my project works fine - the result a .msi file. Is there a way to take that msi file and create an .exe instead?

I tried using a bootstrapper / bundle with `project.BuildMsi();` but I am getting an error and I do not know why.

Thanks I appreciate any kind of help

**Comment by oleg-shilo** [comment, marked as answer]
If you clone the repo you will; find a few examples:

![image](https://github.com/oleg-shilo/wixsharp/assets/16729806/bde87aab-44dc-497b-b706-66fb5369a50b)

However, if you use Visual Studio and WixSharp templates then you can simply create the bootstrapper project and you can build it right away. 

You probably already guessing :) that VS templates offer you the shortest path to success.
